build/buildconstants/params_interop.go

```diff
@@ -1,25 +1,22 @@
 //go:build interopnet
 // +build interopnet
 
-package build
+package buildconstants
 
 import (
+	"log"
```

Comment:
```suggestion
```

`log.Panicf` is referencing the wrong `log` in `getUpgradeHeight` only in this file and not in the rest.